internal/explorer.point/src/context/AppContext.js

```diff
@@ -18,22 +18,36 @@ const AppContext = createContext(defaultContext);
 export const useAppContext = () => useContext(AppContext);
 
 export const ProvideAppContext = ({ children }) => {
-    const [walletIdentity, setWalletIdentity] = useState();
+    const [isLoading, setIsLoading] = useState(true);
+    const [walletIdentity, setWalletIdentity] = useState('');
+    const [identityNetwork, setIdentityNetwork] = useState('');
+    const [publicKey, setPublicKey] = useState('');
+    const [walletAddr, setWalletAddr] = useState('');
     const [walletError, setWallerError] = useState();
     const [, setLocation] = useLocation();
 
     const fetchData = async () => {
         try {
-            const {
-                data: { address },
-            } = await window.point.wallet.address();
-            const {
-                data: { identity },
-            } = await window.point.identity.ownerToIdentity({ owner: address });
-
-            setWalletIdentity(identity);
+            let identityData = {};
+            if (window.point.identity.me) {
```

Comment:
This is a new method I added in a PR in **PointSDK**. It falls back to a _fetch_ request directly to the API so as to not couple both set of changes. This will work even if we don't deploy the changes in PointSDK.